samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -68,7 +78,7 @@ cd samples/msal-node-samples/standalone-samples/ElectronSystemBrowserTestApp
    1. Replace the string `Enter_the_Cloud_Instance_Id_Here` with `https://login.microsoftonline.com`.
    1. Replace the string `Enter_the_Graph_Endpoint_Here`. with `https://graph.microsoft.com/v1.0`.
 
-1. Open the [.authConfig.json](./redirect/public/config/authConfig.json) file and provide the required configuration value
+1. Open the [successTemplate.html](./public/successTemplate.html) file and provide the required configuration value
```

Comment:
Can you go into a little more detail on what is required here?

Reply:
+1

Reply:
@tnorling @derisen I added an explanation in the readme and a comment in the successTemplate.html file. We can change the value to any unique string if it is still confusing.